src/features/targeting/contexts.js

```diff
@@ -5,6 +5,15 @@ import { type FtuxContextValue, type TargetingContextValue } from './types';
 
 const FtuxContext = createContext<FtuxContextValue>({});
 
-const TargetingContext = createContext<TargetingContextValue>({});
+const defaultTargetingContextValue = {
```

Comment:
is this temporary? Maybe I am missing something, but how do these values get passed in from parent app? Would it make sense to expose a function which creates a context given the params?